hq/app/config/LoggingConfig.scala

```diff
@@ -0,0 +1,42 @@
+package config
+
+import java.net.InetSocketAddress
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.util.EC2MetadataUtils
+import model.{DEV, Stage}
+import play.api.Configuration
+
+case class LoggingConfig (
+  stack: String,
+  stage: Stage,
+  app: String,
+  instanceId: String,
+  region: Regions,
+  streamName: Option[String],
+  stsRole: Option[String]
+) {
```

Comment:
Interesting view point. The intention here is to create a logical grouping, `localLogShippingEnabled` doesn't make sense in any context other than logging configuration. Maybe a better pattern would be for these to be properties of the case class and then instantiated at the edge, rather than within the case class.